core/src/main/java/hudson/model/DownloadService.java

```diff
@@ -72,67 +72,20 @@
  * @author Kohsuke Kawaguchi
  */
 @Extension
-public class DownloadService extends PageDecorator {
+public class DownloadService {
 
     /**
      * the prefix for the signature validator name
      */
     private static final String signatureValidatorPrefix = "downloadable";
     /**
      * Builds up an HTML fragment that starts all the download jobs.
+     *
+     * @deprecated browser-based download has been disabled
      */
+    @Deprecated
```

Comment:
OK, though I would suggest just deleting the method as I can imagine no reason a plugin might have been calling it.

Reply:
+1 on deleting I also can't imagine a plugin calling this =/

Reply:
Easy enough to do afterwards, would rather not hold this back longer for that.